src/Integration.Vsix/Resources/Strings.resx

```diff
@@ -277,4 +277,17 @@
   <data name="TextBufferIssueTracker_ProjectGuidError" xml:space="preserve">
     <value>Failed to calculate project guid of file {0}, error occurred: {1}</value>
   </data>
+  <data name="SaveSharedConnectionCommand_SaveSuccess_Message" xml:space="preserve">
+    <value>SonarLint saved the binding configuration to .sonarlint folder in repo root.
```

Comment:
```suggestion
    <value>SonarLint saved the binding configuration to .sonarlint folder (default location - your git repository root, unless specified otherwise)
```

Reply:
But I still think changing the interface to return file path (or using the SharedFolderProvider in the command to find it, if you really don't want to change the interface) would be better than this vague message

Reply:
I'll add a card and maybe tackle it if we have time. I think trying to finish the migration now is more important.